app/controllers/blogs_controller.rb

```diff
@@ -47,6 +48,11 @@ def set_blog
     redirect_to root_path
   end
 
+  def stash_recaptcha_keys
```

Comment:
`stash_recaptcha_keys` appears to be duplicated in a few controllers. When there's duplicate code in a controller, there's a few options for DRYing it up, including moving it to a model, move it to a helper, and move it to the parent class `ApplicationController`.

This responsibility aligns well with `RecaptchaHelper` so consider refactoring this to move this there. Something like:

```rb
module RecaptchaHelper

  # Named public but maybe its private?
  def recaptcha_public_key_v2
    ENV['RECAPTCHA_V2_SITE_KEY']
  end

  def recaptcha_public_key_v3
    ENV['RECAPTCHA_V3_SITE_KEY']
  end
end
```

Those methods are then available to all view templates.